pkg/apis/serving/v1alpha1/kfservice_framework_custom.go

```diff
@@ -21,7 +21,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
-func (c *CustomSpec) CreateModelServingContainer(modelName string) *v1.Container {
+func (c *CustomSpec) CreateModelServingContainer(modelName string, config *FrameworksConfig) *v1.Container {
```

Comment:
Thinking about this, maybe we don't have to do this right now, but we should definitely expose model name to the users container. Perhaps via an environment variable? In fact, we might want to pass a whole suite of well known env vars to the user's container.